consul-application-aware-intentions/intro.md

```diff
@@ -2,6 +2,13 @@
 
 In this hands-on lab, you will operate a Consul datacenter, deployed using Docker, to configure application aware intentions.
```

Comment:
Is this intro text for both the beginning and finish part actually what you want? It doesn't feel like it goes with the list below.